definitions.py

```diff
@@ -0,0 +1,7 @@
+import os
+
+# Path to the root of the project
+ROOT_DIR = os.path.dirname(os.path.abspath(__file__))
+
+# Path to the environment variables
+CONFIG_PATH = os.path.join(ROOT_DIR, 'configuration.conf')
```

Comment:
I convention for the configuration file that I had in my mind was something like `robyn.env`. What do you think about it?

Reply:
i think robyn.env is great i've changed it to that